packages/tools-iottwinmaker/package.json

```diff
@@ -3,30 +3,58 @@
   "private": true,
   "version": "3.0.0",
   "engines": {
-    "node": "^16"
+    "node": ">=16",
+    "npm": ">=9"
   },
-  "main": "./dist/cjs/index.js",
+  "main": "index.js",
   "module": "./dist/es/index.js",
```

Comment:
should remove this line

Reply:
actually discard this comment, when i saw you were no longer pointing to the `dist` folder I had thought you altered the build system